test/journey/TestJourneySpec.scala

```diff
@@ -55,6 +57,43 @@ class TestJourneySpec extends AnyFreeSpec with JourneyHelpers with ModelGenerato
       )
   }
 
+  "test event1 journey (member), payment nature is benefits paid early" in {
+
+    val membersDetails = arbitrary[MembersDetails].sample
+
+    startingFrom(EventSelectionPage)
+      .run(
+        submitAnswer(EventSelectionPage, Event1),
+        submitAnswer(HowAddUnauthPaymentPage, Manual),
+        submitAnswer(WhoReceivedUnauthPaymentPage, Member),
+        next,
+        submitAnswer(MembersDetailsPage, membersDetails.get),
+        submitAnswer(DoYouHoldSignedMandatePage, true),
+        submitAnswer(ValueOfUnauthorisedPaymentPage, true),
+        submitAnswer(SchemeUnAuthPaySurchargeMemberPage, false),
+        submitAnswer(pages.event1.PaymentNaturePage, BenefitsPaidEarly),
+        submitAnswer(BenefitsPaidEarlyPage, ""),
+        pageMustBe(IndexPage)
+      )
+  }
+
+  "test event1 journey for unauthorised payment less than 25%" in {
+
+    val membersDetails = arbitrary[MembersDetails].sample
```

Comment:
we have same test already

"test event1 member-journey for unauthorised payment less than 25%"
I couldn't see the diff ?

Reply:
I don't think we need to keep repeating earlier stages of journey in this spec file. We can just use startingFrom with a given page and continue from there. So for pages like the nature of payment pages we can just say startingFromstartingFrom(PaymentNaturePage). No need for storing subjourneys in vals or memberDetails etc.

Reply:
Removed duplicate in next push.